scraper/modules/subjects_scraper.py

```diff
@@ -234,18 +236,23 @@ def scraper(driver: WebDriver, course_name: str, short_names, master: bool = Fal
                     f"\t\t\033[93m\033[1mWARNING:\033[0m Was not possible to find out \033[4m{subject_name}\033[0m's subject ID. Using 0 as default.")
             # =====================
 
-            if subject_id in short_names.keys():
-                short_name = short_names[subject_id]["short_name"]
+            if str(subject_id) in short_names.keys():
+                short_name = short_names[str(subject_id)]["short_name"]
```

Comment:
Can you also display a warning on the console when this case happens? That way, whoever is running the scraper will know that there is a subject with a short name that needs to be replaced.

Reply:
This is not for prevent repeated short names. This is to get the short names from a previous file that calendarium initially used, from a commit before the scraper, trying to preserve the old short names.

I just did some small improvements with types.

That fix that you want will be done on other PR.

Reply:
This is not for prevent repeated short names. This is to get the short names from a previous file that calendarium initially used, from a commit that exists before the scraper, trying to preserve the old short names.

I just did some small improvements with types.

That fix that you want will be done on other PR.